org-eclipse-che-jdt-ls-extension/jdt.ls.extension.api/src/main/java/org/eclipse/che/jdt/ls/extension/api/dto/navigation/ImplementationsDescriptor.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2012-2017 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.jdt.ls.extension.api.dto.navigation;
+
+import java.util.List;
+import org.eclipse.lsp4j.SymbolInformation;
+
+public class ImplementationsDescriptor {
+
+  private String memberName;
```

Comment:
A "member" in jdt parlance is a field, method, etc. But not necessarily a type. I think what we have here is the name of the interface we're looking for? If so, we should call it "interfaceName", or "searchedElement" or the likes.